src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,44 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("요구사항1 - \"1,2\" 문자열이 주어지고 , 로 split했을 때 1과 2로 분리된 배열을 반환한다.")
+    void split1() {
+        String[] result = "1,2".split(",");
+
+        assertThat(result).contains("1", "2");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?